pkg/apis/core/helper/helpers.go

```diff
@@ -133,6 +133,21 @@ func TaintsHave(taints []core.SeedTaint, key string) bool {
 	return false
 }
 
+// SeedSettingExcessCapacityReservationEnabled returns true if the 'excess capacity reservation' setting is enabled.
+func SeedSettingExcessCapacityReservationEnabled(settings *core.SeedSettings) bool {
+	return settings != nil && settings.ExcessCapacityReservation != nil && settings.ExcessCapacityReservation.Enabled
```

Comment:
```suggestion
	return settings == nil || settings.ExcessCapacityReservation == nil || settings.ExcessCapacityReservation.Enabled
```

My point is that by default the excess capacity is enabled, thus this method should return `true` even when excess capacity is not explicitly enabled. 

The same comment is applicable for visible and dns.

Reply:
Thanks for being so careful, indeed!